modules/oxide/tasks/js-skins.js

```diff
@@ -20,7 +20,15 @@ module.exports = function (grunt) {
           grunt.log.error(err);
           return done(false);
         }
-        const content = `tinymce.Resource.add('${file.replace(/\.min.css$/, '.css')}', \`${data.split("\n")[0]}\`)`
+
+        const fileLines = data.split("\n");
+        // Check if the minified CSS file has more than 2 lines
+        // Line 1: The minified CSS content
+        // Line 2: The sourcemap comment, eg: (/*# sourceMappingURL=content.min.css.map */)
```

Comment:
Oh _that's_ what this is doing, good catch. We definitely need a new tech debt task to improve this validation instead of guessing with line numbers.

Reply:
I've logged a ticket: https://ephocks.atlassian.net/browse/TINY-11881 to improve this validation and also the tool that we use. I agree, but hopefully, this is reliable enough until we address the root cause.